### DISCUSSION TITLE
Archlinux

### DISCUSSION THREAD

**Discussion by kamalkech** [go-task/task#869] [Q&A]
taskfile-git not found! are you change the app for archlinux ?? if yes plz update official documentation

**Comment by andreynering** [comment]
Hey @kovetskiy,

Do you know what happened here? Seems that the package disappeared.

I just found https://aur.archlinux.org/packages/go-task but I'm not sure if it was renamed or this one is another package.

**Reply by kamalkech** [reply]
me too currently i am using go-task

**Reply by kovetskiy** [reply]
@andreynering hey, I'm no longer maintaining the package. There was some issues like conflicts with community/task package, if I remember correctly, I didn't have time to fix it, but then a person contacted me and asked for transferring package ownership to take care of the issues.

**Reply by andreynering** [reply]
Hey @carlsmedstad,

Seems that you maintain the [go-task](https://aur.archlinux.org/packages/go-task) and [go-task-bin](https://aur.archlinux.org/packages/go-task-bin) packages, right?

I'm going to update the documentation mentioning the new package. Which one would be preferred? I believe `go-task-bin` which seems to take the official binaries, but I'd like an opinion as I'm not an Arch guy.

Thanks!

**Reply by carlsmedstad** [reply]
Hey!

Yeah, I'd go for `go-task-bin` in the docs. Binary AUR packages are more popular for these kinds of tools in my experience.

Glad to be of help :)

**Reply by andreynering** [reply]
Thank you @carlsmedstad!

Just updated the documentation at 081d878f86f696565eb8042928099e833b3d5cac.
